mentors/go/k.json

```
@@ -46,5 +46,13 @@
     "link_url": null,
     "avatar_url": null,
     "bio": "Experienced GOpher!"
+  },
+  {
+    "github_username": "KellenWatt",
+    "name": "Kellen Watt",
+    "link_text": null,
+    "link_url": null,
+    "avatar_url": null,
+    "bio": "I love exploring new, old, and exciting languages, but no matter what I find, I always come back to Go!
```

```suggestion
    "bio": "I love exploring new, old, and exciting languages, but no matter what I find, I always come back to Go!"
```
Just missing a closing quote mark.